.github/workflows/update-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Release Branch
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
I think if you do just `tags` without any list items it's equivalent.

Otherwise, if you want to have a pattern there, use `v*`